Casks/i/ibm-aspera-connect.rb

```diff
@@ -1,24 +1,27 @@
 cask "ibm-aspera-connect" do
-  version "4.1.3.93"
-  sha256 "502fe4644ed79d214a9acc33922e55de88c50ddd37dd4072515a1a51c287d617"
+  arch arm: "arm64", intel: "x86_64"
 
-  url "https://d3gcli72yxqn2z.cloudfront.net/connect_latest/v4/bin/ibm-aspera-connect_#{version}_macOS_x86_64.dmg",
-      verified: "d3gcli72yxqn2z.cloudfront.net/"
+  version "4.2.13.820"
+  sha256 arm:   "58b42f814c95168e149491d330acb286920fcc19c581e958168305b78c8aa478",
+         intel: "7dd745e07145a39cfae516b797745541fb3774050e567f98adbe191651b10d1f"
+
+  url "https://download4.boulder.ibm.com/sar/CMA/OSA/0csnb/0/ibm-aspera-connect_#{version}_macOS_#{arch}.pkg"
   name "IBM Aspera Connect"
   desc "Facilitate uploads and downloads with an Aspera transfer server"
   homepage "https://www.ibm.com/aspera/connect/"
 
   livecheck do
-    url "https://d3gcli72yxqn2z.cloudfront.net/connect_latest/v4/connectversions.min.js"
-    regex(/ibm-aspera-connect[._-]v?(\d+(?:\.\d+)+)_macOS_x86_64\.dmg/i)
+    url "https://d3gcli72yxqn2z.cloudfront.net/downloads/connect/latest/versions.js"
+    regex(/ibm-aspera-connect[._-]v?(\d+(?:\.\d+)+)_macOS_x86_64\.pkg/i)
```

Comment:
```suggestion
    regex(/ibm-aspera-connect[._-]v?(\d+(?:\.\d+)+)_macOS_#{arch}\.pkg/i)

Reply:
We should also be able to replace the separators:

```suggestion
    regex(/ibm-aspera-connect[._-]v?(\d+(?:\.\d+)+)[._-]macOS[._-]#{arch}\.pkg/i)
```

Reply:
I'm unable to find a better option as well.